pandas/tests/resample/test_base.py

```diff
@@ -25,6 +25,27 @@
 from pandas.core.resample import _asfreq_compat
 
 
+@pytest.fixture(scope="module")
+def all_1d_no_arg_interpolation_methods():
```

Comment:
As a fixture you can define it like:

```python
@pytest.fixture(params=["linear", "time", ...])
def all_1d_no_arg_interpolation_methods(request):
    return request.param
```

Reply:
Did not know that one, thanks you @mroeschke 😃 Fixed